_rules/html-has-lang-b5c3f8.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: b5c3f8
-name: HTML has lang attribute
+name: HTML pages have lang attribute
```

Comment:
Why 'pages' instead of 'page'?
The applicability if for the root element of the web page. Also, in 'HTML page languages are valid' you use the singular (but in 'HTML pages have a title' you use the plural - so it's not coherent)
I do prefer the singular form instead of using plural by default, because that leaves us the plural for any instance where we have rules that apply to sets

Reply:
The group consensus on page titles (see #250) is that plural is preferable. "HTML page languages" is also plural, so I think that is consistent.

The other thing to consider is collections of pages. If you are testing WCAG on a collection of pages, all of them have to have a lang attribute. This is for example true for EPUB, which is a single EPUB page, that is a bundle of HTML web pages.

Reply:
I was only able to find support for single tense in #250 (that's also what I prefer). Perhaps there was consensus on a call. No need to drag this.

Reply:
Right you are... :/ I'm not sure where my head was at yesterday. Sorry about that!